src/io/flutter/preview/PreviewView.java

```diff
@@ -548,45 +501,13 @@ private FlutterOutline getOutlineOfPath(@Nullable TreePath path) {
     return getOutlineOfNode(node);
   }
 
-  private int getConvertedFileOffset(int offset) {
```

Comment:
these outline location helpers were moved to the outline coverter helper class as they were needed a bunch of places.